toptal/gitignore.io
# [SAT-433] Measure Field Data in Google Analytics

### Description

Send Core Web Vitals information to Google Analytics

### How to test

- Edit docker-compose-dev.yml and add a valid `GOOGLE_ANALYTICS_UID`
- `docker-compose -f docker-compose-dev.yml up` go to any page and open the [Google Tag Assistant](https://chrome.google.com/webstore/detail/tag-assistant-legacy-by-g/kejbdjndbnbjgmefkgdddjlbokphdefk) and enable tracking from the extension
- Refresh the page, and you should see LCP in the list of events
- Click anywhere on the page and you should see FID also emitted as an event
- Go to another tab and come back to the page and you should see CLS emitted as a events (emitted on visibilityHidden)
